Documentation/guides/building-apps/build-items.md

```diff
@@ -174,6 +174,62 @@ installing app bundles.
 
 This build action was introduced in Xamarin.Android 11.3.
 
+## AndroidKnownDesignerAssemblies
+
+This is an ItemGroup that is used to work around issues when upgrading the old `Resource.designer.cs`
+system to the new Resource Assembly system. This can happen if an assembly is referencing a 
+`Resource.designer.cs` class from anther assembly which was built on an older framework. 
+If you see runtime errors such as 
+
+```
+System.TypeLoadException: 'Could not resolve type with token 010001d8 from typeref (expected class 'Style' in assembly '')'
+```
+
+or build errors such as
+
+```
+error NETSDK1144: Optimizing assemblies for size failed. Optimization can be disabled by setting the PublishTrimmed property to false
+```
+
+or 
+
+```
+Unhandled exception. Mono.Linker.LinkerFatalErrorException: ILLink: error IL1013: Error processing '…/dotnet-sdk-8.0.100-rc.1.23455.8-osx-x64/packs/Microsoft.Android.Sdk.Darwin/34.0.0-rc.1.432/targets/../PreserveLists/Mono.Android.xml'.
```

Comment:
I saw this in RC1.  If I take the [test-maui-17265](https://github.com/xamarin/xamarin-android/tree/test-maui-17265) branch and hack it up so that `@(AndroidKnownDesignerAssemblies)` is empty / the `@(_TrimmerCustomData)` addition is removed -- so that I remove the fix -- I no longer see this IL1013 error. :-(

In fact, my Release build now has *no* errors!

Which is quite surprising to me. :-/

Reply:
argh!

OK, part of the problem is that diagnostic builds ***are not enough***.  You *also* need to set `$(_ExtraTrimmerArgs)`=--verbose!

```sh
~/Downloads/dotnet-sdk-8.0.100-rc.1.23455.8-osx-x64/dotnet build -f net8.0-android \
    -p:Configuration=Release -v:diag -p:_ExtraTrimmerArgs=--verbose
```